medusa/tv/series.py

```diff
@@ -1217,7 +1217,7 @@ def should_refresh_file(cur_status, same_file, check_quality_again, anime, filep
             if new_quality != Quality.UNKNOWN:
                 return True, 'New file has different name from the database but has valid quality.'
             else:
-                return False, 'New file has UNKNOWN quality'
+                return True, 'New file has different name from the database and an UNKNOWN quality.'
```

Comment:
@fernandog shouldn't we make this consistent with the should_replace? As now, when I have an UNKNOWN quality file downloaded locally, it will not upgrade to an "allowed" quality if found.